### ISSUE TITLE
[Major Bug] [FIX IN REPLIES] Dorion can no longer join/make any voice calls of any form (VCs are unusable), rendering a large majority of Discord unaccessible to Dorion users.

### ISSUE THREAD

**Issue by PeaceDevWasTaken** [SpikeHD/Dorion#132]
**Describe the bug**
Describe the problem here.
Discord says Dorion can not join voice calls because the browser is out of date
**To Reproduce**
Join any voice channel in the Dorion client
**Expected behavior**
Describe what should be expected here.
it should just join the voice channel
**Screenshots**

**System Info (please complete the following information):**
 - OS: Windows 10
 - Version: Latest
 - Any other relevant info: 

**Additional context**
Attempting to join a voice channel causes the application to state the web browser is unsupported/out of date and to use opera firefox chrome edge ect as an alternative, essentially making the client unusable in voice calls.

**Comment by SpikeHD** [comment]
This is known and was fixed in 2.0, which is hopefully releasing pretty soon. If you'd like a slightly jank fix in the meantime:

1. Create a `.js` file in your Dorion plugins folder. For example, I'd name it to `temp_voice_fix.js`
2. In that file, paste this code:

```js
  // Set useragent to be Chrome as it is closest to what we actually are
  Object.defineProperty(navigator, 'userAgent', {
    get: () => 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36'
  })
```
3. Enable it and set it to "Preload" in Dorion settings
4. Restart

While it won't break anything, you'll probably want to delete this fix once you update to 2.0.

**Comment by PeaceDevWasTaken** [comment]
Thanks for the fix, but I just ended up downloading the preview 2.0.0 windows exclusive build from your discord server

If anyone else wants it here you go
https://cdn.discordapp.com/attachments/1128911050117488640/1168708610105299034/Dorion_2.0.0_x64_en-US.msi?ex=6552bfb6&is=65404ab6&hm=bd15fd33380da12754102577248c124815b70f661889914465bdab99c50cfa0b&
